tests/js/client/shell/shell-admin-cluster-rebalance.js

```diff
@@ -254,9 +249,10 @@ function clusterRebalanceOtherOptionsSuite() {
 
           result = getRebalancePlan(true, true, true);
           let moves = result.result.moves;
-          assertTrue(moves.length > 0);
-          for (const job of moves) {
-            assertNotEqual(job.to, dbServer.id);
+          if (moves.length > 0) {
```

Comment:
Is the `if` required here? If `moves` is an empty array, the old code should have worked as well, right?

Reply:
Actually not, the for loop wouldn't have any iteration to do if `moves === 0`. I let it more for being explanatory, but I'll remove it because it's unnecessary